src/signature/SigningAccount.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "signature/SigningAccount.h"
+#include "ledger/AccountFrame.h"
+
+namespace stellar
+{
+
+SigningAccount::SigningAccount(AccountID accountID) :
```

Comment:
I prefer as a pattern to pass in a const ref and use the copy constructor (consistent with the rest of the code base), similar to what you're doing with the other constructor `SigningAccount(AccountFrame const& accountFrame)`